docs/operating-scylla/procedures/cluster-management/add-dc-to-existing-dc.rst

```diff
@@ -1,13 +1,30 @@
 Adding a New Data Center Into an Existing ScyllaDB Cluster
 ***********************************************************
 
+.. caution::
+
+    The procedure described below applies to clusters where Raft-based topology 
+    is enabled. Raft-based topology is automatically enabled in new clusters.
+
+    If you've upgraded an existing cluster from version 5.4, ensure that you 
+    :doc:`manually enabled Raft-based topology </upgrade/upgrade-opensource/upgrade-guide-from-5.4-to-6.0/enable-raft-topology>`.
+    Without Raft-based topology enabled, you must consider the following limitations 
+    while following the procedure:
+
+    * You can only bootstrap one node at a time.
+    * If the node starts bootstrapping but fails in the middle, for example, due to 
+      a power loss, you can retry bootstrap by restarting the node. If you don't want to
+      retry, or the node refuses to boot on subsequent attempts, consult the 
+      :doc:`Handling Membership Change Failures </operating-scylla/procedures/cluster-management/handling-membership-change-failures>`
+      document. 
+    * You must restart ScyllaDB nodes one by one, not in pararell, in both the new and existing datacenters.
```

Comment:
It is still important to restart nodes one by one, because shutting down multiple nodes may cause application unavailability!

For example, consider the most common setup, RF=3 table, application performing CL=LOCAL_QUORUM reads/writes in the existing DC.

Rolling restart is safe to do without availability loss, because there are always 2 out of 3 replicas available. But if you shut down 2 nodes at a time, there will be only 1 replica available, and the LOCAL_QUORUM queries will start failing!


This has nothing to do with topology changes.

Reply:
There had to be some miscommunication at some point --- most procedures that involve node restarts should be unaffected by Raft-based topology.

In fact, I am only aware of **additional limitations** for node **restarts** in Raft-based topology, I'm not aware of getting rid of limitations. (The additional limitation I have in mind is that now quorum is required if we restart a node with different number of shards. There was no such limitation pre-Raft-based-topology)